examples/flutter_view/android/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="app_name">Flutter View</string>
+  <string name="title">Flutter Application</string>
+  <string name="increment">+</string>
+  <string name="button_tap">Flutter button tapped 0 times.</string>
```

Comment:
As discussed with Mikkel, I'll leave it as is: We can use plurals but AFAICS it adds duplicate code elsewhere. Also leaving it like this keeps it symmetric with how it is done on iOS.